ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLFlowAgentRunner.java

```diff
@@ -287,13 +287,18 @@ Map<String, String> getToolExecuteParams(MLToolSpec toolSpec, Map<String, String
                 executeParams.put(key, params.get(key));
             }
         }
+        // Override all parameters in tool config to tool execution parameters as the config contains the static parameters.
+        if (toolSpec.getConfigMap() != null && !toolSpec.getConfigMap().isEmpty()) {
```

Comment:
rather than putting same piece of code, can we try to use a common method `getToolExecuteParams` for both MLFlowAgentRunner & MLConversationalFlowAgentRunner?